### PR TITLE
Updated Commons Collections lib and moved to Target platform

### PR DESCRIPTION
**Pull request by MarcelS4**
Updated the commons-collections library and moved it to the target
platform file. The necessary code changes were made to fix compile
issues and as a result commons-primitives was removed as it was no
longer necessary.

This change was necessary to avoid security vulnerability

### CODE REVIEW

**Comment by MarcelS4** [comment]
Dependencies Update #252

**Comment by wimjongman** [comment]
Marcel, All, please assign milestones and connect PR's to an issue so that we have our new and noteworthy, and release documentation up to date.

image

**Comment by vogella** [comment]
@MarcelS4 this seems to contains also other changes, please "cleanup" the commit so that it only contains changes related to the lib change.

**Comment by MarcelS4** [comment]
Ran the WindowBuilderTests test suite and no errors or failing tests appear

**Comment by vogella** [comment]
Great. Thanks.

**Comment by vogella** [comment]
Still a lot of unrelated format changes. Can you undo them?

**Comment by MarcelS4** [comment]
I'll look into it. but I just copied and pasted the previous commit's line by line so I dont know how they got in there, but will look into it and change it

**Comment by vogella** [comment]
I handle the formatting changes via a new pull request

File: org.eclipse.wb.core/src/org/eclipse/wb/internal/core/model/property/table/PropertyTable.java [hunk 1]
```diff
@@ -148,30 +148,30 @@ public PropertyTable(Composite parent, int style) {
    * Adds listeners for events.
    */
   private void hookControlEvents() {
-    addListener(SWT.Dispose, new Listener() {
```
Comment (vogella) [inline org.eclipse.wb.core/src/org/eclipse/wb/internal/core/model/property/table/PropertyTable.java:151]: Will move this to another PR

File: org.eclipse.wb.core/src/org/eclipse/wb/internal/core/utils/external/ExternalFactoriesHelper.java [hunk 2]
```diff
@@ -111,15 +111,15 @@ private static synchronized void clearCache(String pointId) {
...
-      new IRegistryChangeListener() {
-        @Override
-        public void registryChanged(IRegistryChangeEvent event) {
-          for (IExtensionDelta extensionDelta : event.getExtensionDeltas()) {
-            String pointId = extensionDelta.getExtensionPoint().getUniqueIdentifier();
-            clearCache(pointId);
-          }
+		  new IRegistryChangeListener() {
+      @Override
+      public void registryChanged(IRegistryChangeEvent event) {
```
Comment (vogella) [inline org.eclipse.wb.core/src/org/eclipse/wb/internal/core/utils/external/ExternalFactoriesHelper.java:116]: Will move to another PR

**Review by vogella — CHANGES_REQUESTED** [review]

**Comment by vogella** [comment]
Thanks @MarcelS4

**Pull request closed without merging**